pom.xml

```diff
@@ -5,18 +5,20 @@
     <groupId>io.split.client</groupId>
     <artifactId>java-client-parent</artifactId>
     <version>2.0.3-rc2</version>
-    <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>1.7.21</version>
```

Comment:
The root POM was declaring this version, but both children were using the older `1.7.12` (the child POM takes precedence). To avoid a version upgrade as part of this change I decided to downgrade in the root POM to match the version that was actually being used. Feel free to edit if you'd like to bump.